crypto/evp/p_lib.c

```diff
@@ -688,6 +688,38 @@ ENGINE *EVP_PKEY_get0_engine(const EVP_PKEY *pkey)
 # endif
 
 # ifndef OPENSSL_NO_DEPRECATED_3_0
+static void detect_low_level_engine(EVP_PKEY *pkey)
+{
+    switch (pkey->type) {
+        case EVP_PKEY_RSA:
```

Comment:
indentation is off for the switch's whole body.

Reply:
Fixup pushed